src/CLI/CLI.csproj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <AssemblyName>AxeWindowsScanner</AssemblyName>
```

Comment:
How about naming this AxeWindowsCLI? It seems more informative than "scanner".

Reply:
Done